WINDOWS_HOWTO.md

```diff
@@ -0,0 +1,49 @@
+**COMPILATION AND INSTALLATION OF UTILITIES**
+
+_For support of systems with more than _**_64_**_ logical cores you need to compile all binaries below in “_**_x64_**_” platform mode (not the default “_**_Win32_**_” mode)._
+
+Command-line utility:
+
+1.Compile the windows MSR driver (msr.sys) with Windows DDK Kit (see the sources in the WinMSRDriver directory). For Windows 7 you have to sign the msr.sys driver additionally ([http://msdn.microsoft.com/en-us/library/ms537361(VS.85).aspx](%22http://)).
+
+2.Build the pcm.exe utility in the PCM_Win using Microsoft Visual Studio
+
+3.Copy the msr.sys driver and pcm.exe into a single directory
+
+4.Run pcm.exe utility from this directory
+
+For Windows 7 and Windows Server 2008 R2 the PCM utilities need to be run as administrator:
+
```

Comment:
could you add the picture from the original RTF to the repository and refer it here?

Reply:
Done.

Reply:
> Done.

I don't see any update for your change. Something went wrong?

Reply:
@jrmuizel I like your change. Is it possible to submit a revised change?